backend/main.py

```diff
@@ -88,59 +100,44 @@ async def retrieve_available_lights():
 @fast_app.post("/selected-lights")
 def store_selected_lights(lights: List[str]):
     if not lights:
-        raise HTTPException(status_code=400, detail="Selected lights list is required")
+        raise HTTPException(status_code=400, detail='\"lights\" list is required.')
 
     try:
-        redis_client.set("lights", ",".join(lights))
+        redis_client.set(constants.REDIS_SELECTED_LIGHTS_KEY, ",".join(lights))
```

Comment:
Here we store the selected lights as a comma separated string. See comment below.

Reply:
Based on this, 

This option:
(if we want only change the task's lights to white) access that task object's .args on the backend and get the lights to be changed that way,

then do we need the redis selected lights key?